### DISCUSSION TITLE
My codespaces was 8 cores but it not available. It shows 2 cores 8GB RAM. I have a student developer pack. Please help with this.

### DISCUSSION THREAD

**Discussion by imostafizur** [community/community#44714] [Codespaces]
### Select Topic Area

Product Feedback

**Comment by 8Gitbrix** [comment]
Here's the docs which explain how to change the machine type for your codespace: https://docs.github.com/en/codespaces/customizing-your-codespace/changing-the-machine-type-for-your-codespace#about-machine-types 

But, from the docs you should note that
> These machine types offer a choice of hardware specifications ranging from 2 cores to 32 cores, although the full range of machine types may not always be available.

**Reply by AndnixSH** [reply]
is there any reason why 8 cores or more are not available, even with pro account?

**Reply by MauScheff** [reply]
Why would it not be available if it's advertised as part of the benefits on the landing page?

**Reply by Seb0927** [reply]
:(

**Reply by MauScheff** [reply]
I figured that it is available, just had to stop / delete the other running machines, or something like that, can't remember right now.
